### PR TITLE
feat(repositories): Add conection options to repository manifest

### LINKED ISSUE #57: Feaure request (https proxy): Repository --> proxy --> "Use the Nexus truststore" is not implemented.
Hi guys,
Using Nexus 3.23.0 can't find this option to be either enabled or disabled for the locations which use https protocol:

image

Don't see it in a "proxy" code configuration section for the repository as well.
It would be nice to have it added. 
Thank you a lot in advance! :)

Kindest regards,
Taras.

### PR DESCRIPTION
**Pull request by anmoel**
fixes #57

### CODE REVIEW

File: nexus/resource_repository_apt_proxy_test.go [hunk 1]
```diff
@@ -14,7 +14,11 @@ func testAccResourceRepositoryAptProxy() nexus.Repository {
 		Distribution: "bionic",
 		Flat:         true,
 	}
+	useTrust := true
```
Comment (stevie-) [inline nexus/resource_repository_apt_proxy_test.go:17]: Better naming? `useTrustSore`

**Review by stevie- — APPROVED** [review]
/lgtm

**Review by sebbrandt87 — DISMISSED** [review]

**Review by stevie- — APPROVED** [review]
/lgtm

File: nexus/resource_repository_apt_proxy_test.go (hunk 1 above)
Comment (anmoel) [reply nexus/resource_repository_apt_proxy_test.go]: done

**Pull request merged**